packages/strapi-plugin-upload/admin/src/components/CardControl/Wrapper.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  display: flex;
+  width: 24px;
+  height: 24px;
+  margin-right: 5px;
+  background-color: #ffffff;
+  border: 1px solid #e3e9f3;
```

Comment:
Agree for the `background-color` however regarding the `border` it doesn't exist in the theme and since I am colourblind it's really hard for me to put a name on it so I let you update it in one of your PRs!